tests/test_pandas_extension.py

```diff
@@ -393,6 +393,11 @@ def test_factorize_equivalence(self, data_for_grouping, na_sentinel):
                 self, data_for_grouping, na_sentinel
             )
 
+    @pytest.mark.parametrize("box", [pd.Series, lambda x: x])
+    @pytest.mark.parametrize("method", [lambda x: x.unique(), pd.unique])
+    def test_unique(self, data, box, method):
```

Comment:
Why do you redefine this here? If it is part of the base pandas class, there is not need to define it here.

Reply:
You're right, I removed it.